archive/frames/models.py

```diff
@@ -34,88 +36,159 @@ class Frame(models.Model):
     basename = models.CharField(max_length=1000, db_index=True, unique=True)
     area = models.PolygonField(geography=True, spatial_index=True, null=True, blank=True)
     related_frames = models.ManyToManyField('self', blank=True)
+    observation_date = models.DateTimeField(
```

Comment:
Is the plan to add the new fields, test that things work, then delete the old fields? As opposed to replacing them.

Reply:
Yeah, I think this way if there was an issue, it will be easier to revert (since the old fields still exist, without a migration). Then when we have deployed, migrated, and tested, we can deploy another update to remove the old fields completely